sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/EmailGenerator.groovy

```diff
@@ -0,0 +1,52 @@
+package life.qbic.samplenotificator.components
+
+import life.qbic.business.subscription.Subscriber
+
+import java.util.concurrent.TimeUnit
+
+/**
+ * Class responsible for generating an sending an email from the provided notification and subscriber information
+ *
+ * EmailGenerator converts the notification created in the CreateNotification Use case into an emailBody
+ * The contactInformation such as e.g. the emailAddress of the recipient is retrieved from the associated provided Subscriber
+ * The email is then send via a commandline call to the mailutils tool(@link <a href=https://mailutils.org/>mailutils</a> via a ProcessBuilder
+ *
+ * @since: 1.0.0
+ *
+ */
+class EmailGenerator {
+
+    private String subject = "Project Update"
+    Map<Subscriber, String> notificationPerSubscriber
+
+    EmailGenerator(Map<Subscriber, String> notificationPerSubscriber) {
+        this.notificationPerSubscriber = notificationPerSubscriber
+    }
+
+    /**
+     * Triggers the creation and submission of each email
+     * containing the provided notifications to the provided subscribers
+     */
+    void initializeEmailSubmission(){
+        notificationPerSubscriber.each {
+            //ToDo Replace this with subscriber mail
+            sendEmail("Steffen.greiner@uni-tuebingen.de", it.value)
```

Comment:
I'd appreciate it greatly if we changed this emailadress to a more generic email address for testing purposes
```suggestion
            sendEmail("Developer@portal-testing.qbic.uni-tuebingen.de", it.value)
```